src/ast/passes/codegen_llvm.cpp

```diff
@@ -3561,6 +3570,14 @@ void CodegenLLVM::probereadDatastructElem(Value *src_data,
         b_.SetInsertPoint(pred_true_block);
       }
     }
+    else if (data_type.is_btftype)
+    {
+      expr_ = b_.CreateDatastructElemLoad(
+          elem_type,
+          b_.CreateIntToPtr(src, dst_type->getPointerTo()),
+          true,
+          data_type.GetAS());
+    }
```

Comment:
This is exactly the same call as on lines 3545-3549, can we merge them?

Reply:
Ok.